### PR TITLE
Code Buddy - Setup

### LINKED ISSUE #1: 0.5pt - Set up of the project
Use correct Gitflow
Create the Code Buddy app
Add linters

### PR DESCRIPTION
**Pull request by Donard97**
**I opened this PR for you to review if my project is as requested 👀** 

_In this milestone, I had to do only the setup for the Code Buddy backend._ 💯 💯 💯 

### Project requirements that are done:

> - Use correct Gitflow ☑️ 
> - Create the Code Buddy app ☑️ 
> - Add linters ☑️ 

<hr>

_Feel free to review my project 👁️‍🗨️ and if you have any questions please ask me! ✌️_

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/ror/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (ahzia) [inline .github/workflows/linters.yml]: It is an API  Backend App So we don't need styleint, Comment the linters configuration for styleint

File: .stylelintrc.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "extends": ["stylelint-config-standard"],
+  "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+  "rules": {
+    "at-rule-no-unknown": null,
+    "scss/at-rule-no-unknown": true,
+    "csstree/validator": true
+  },
+  "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css"]
+}
```
Comment (ahzia) [inline .stylelintrc.json]: Remove This file, It is an API Backend App So we don't need styleint,

**Review by ahzia — APPROVED** [review]
**STATUT: APPROVED 🎉🍾**
Hello @Donard97  👋

**[Optional] You have added some frontend assets, this project is an API Backend Project and we don't need JS assets.**

- Remove linters Configuration (see the inline comments)
- Remove Package.json and Package-lock.json
- Remove all Frontend Related assets (seeing your commit history you can do by reverting your last commit)

**[Optional] Configure CORS so we can do API requests from anywhere..**

 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: .github/workflows/linters.yml (hunk 1 above)
Comment (Donard97) [reply .github/workflows/linters.yml]: Thank you for the feedback! I commented linters configuration for stylelint. ✌️

File: .stylelintrc.json (hunk 2 above)
Comment (Donard97) [reply .stylelintrc.json]: Thank you for letting me know. I deleted stylelintrc file. 💪

**Pull request merged**